lsp/lsp.cpp

```diff
@@ -460,8 +463,16 @@ class LSP {
 };
 
 
-int main() {
-  LSP lsp;
-  // Process requests until something goes wrong 
-  lsp.processRequests();
+int main(int argc, const char **argv) {
+  std::string stdLib;
+  if (argc == 2) {
+    stdLib = argv[1];
+  } else {
+    stdLib = find_execpath() + "/../../share/wake/lib";
+  }
+  if (access((stdLib + "/core/boolean.wake").c_str(), F_OK) != -1) {
+    LSP lsp(stdLib);
+    // Process requests until something goes wrong
+    lsp.processRequests();
+  }
```

Comment:
You probably should print an error message and return non-zero if the access fails.

Reply:
Whatever. Fix it in a follow-up. This PR is green and already very large.